test/fixtures/dialects/snowflake/snowflake_alter_warehouse.sql

```diff
@@ -21,3 +21,6 @@ alter warehouse LOAD_WH SET MAX_CONCURRENCY_LEVEL = 1;
 alter warehouse LOAD_WH UNSET STATEMENT_QUEUED_TIMEOUT_IN_SECONDS;
 alter warehouse LOAD_WH UNSET WAREHOUSE_SIZE;
 alter warehouse LOAD_WH UNSET WAREHOUSE_SIZE, WAIT_FOR_COMPLETION;
+
+ALTER WAREHOUSE SET WAREHOUSE_SIZE='X-LARGE';
+alter warehouse set warehouse_size=medium
```

Comment:
I would usually add the ending semicolon but will just leave it for now so we don't have to rehash.